include/aws/http/http.h

```diff
@@ -54,6 +54,7 @@ enum aws_http_errors {
     AWS_ERROR_HTTP_STREAM_IDS_EXHAUSTED,
     AWS_ERROR_HTTP_INVALID_FRAME_SIZE,
     AWS_ERROR_HTTP_COMPRESSION,
+    AWS_ERROR_HTTP_FLOW_CONTROL,
```

Comment:
this doesn't seem very .... descriptive?

Reply:
Agreed.

This maps will map to [FLOW_CONTROL_ERROR](https://httpwg.org/specs/rfc7540.html#FLOW_CONTROL_ERROR) from HTTP/2.

How about:
`AWS_ERROR_HTTP_FLOW_CONTROL_WINDOW`?
`AWS_ERROR_HTTP_FLOW_CONTROL_VIOLATION`?
`AWS_ERROR_HTTP_FLOW_CONTROL_WINDOW_ERROR`?

Reply:
removing for now.
deal with it in another PR